sections/introduction/mobility.tex

```diff
@@ -0,0 +1,8 @@
+\section{Mobility}\label{sec:mobility}
+As a part of the study regulation it is emphasized that the system needs to be mobile.
+The essentiality of the project idea as described in \autoref{sec:projectidea}, is that the players are mobile and easily can use their smartphone and virtual reality headset as the viewable device to view the game.
+To achieve this form of mobility it requires us to handle wireless communication between users and by calculating the location of the players by the use of sensors.
+\\\\
+We also desire to have the entire system as being mobile, so that you would be able to set up the game at most locations, either indoors or outdoors.
+This can however be more difficult, as some type of location-based service has to be used, as well as the need for connection between users to communicate.
+Setting up the game at other locations also requires more knowledge about the game from the user, than the regular player otherwise would know, as the person has to set up the sensors at correct positions to measure the distance between the components.
```

Comment:
```suggestion
Setting up the game at other locations also requires more knowledge about the game from the user, than the regular player would have, as the user has to set up the anchors at correct positions to measure the distance between the anchors.
```

Reply:
Vi har ikke introduceret anchors, så derfor benytter jeg sensors i stedet for anchors